### DISCUSSION TITLE
showCreateRelationButton doesn't seem to work with MorphToMany Relationships

### DISCUSSION THREAD

**Discussion by iHwy** [laravel/nova-issues#4574] [Q&A]
From the docs:

You may also create related many-to-many relationships from the "attach" and "update attached" pages. To enable this feature, invoke the showCreateRelationButton when defining a BelongsToMany or MorphToMany relationship:

BelongsToMany::make('Roles')->showCreateRelationButton(),

Was this removed, never implemented or am I misunderstanding?

**Comment by AnastasiaDolgopolova** [comment]
похоже это все еще не работает..
 или я это неправильно использую?

**Reply by AnastasiaDolgopolova** [reply]
проблема решилась при помощи https://github.com/dillingham/nova-attach-many
